docs/api/jdk8/platform/cuda/com/ibm/cuda/package-tree.html

```diff
@@ -125,12 +125,12 @@ <h2 title="Enum Hierarchy">Enum Hierarchy</h2>
 <li type="circle">java.lang.<a href="https://docs.oracle.com/javase/8/docs/api/java/lang/Enum.html?is-external=true" target="_blank" title="class or interface in java.lang"><span class="typeNameLink">Enum</span></a>&lt;E&gt; (implements java.lang.<a href="https://docs.oracle.com/javase/8/docs/api/java/lang/Comparable.html?is-external=true" target="_blank" title="class or interface in java.lang">Comparable</a>&lt;T&gt;, java.io.<a href="https://docs.oracle.com/javase/8/docs/api/java/io/Serializable.html?is-external=true" target="_blank" title="class or interface in java.io">Serializable</a>)
 <ul>
 <li type="circle">com.ibm.cuda.<a href="../../../com/ibm/cuda/CudaJitInputType.html" title="enum in com.ibm.cuda"><span class="typeNameLink">CudaJitInputType</span></a></li>
-<li type="circle">com.ibm.cuda.<a href="../../../com/ibm/cuda/CudaDevice.CacheConfig.html" title="enum in com.ibm.cuda"><span class="typeNameLink">CudaDevice.CacheConfig</span></a></li>
-<li type="circle">com.ibm.cuda.<a href="../../../com/ibm/cuda/CudaDevice.Limit.html" title="enum in com.ibm.cuda"><span class="typeNameLink">CudaDevice.Limit</span></a></li>
-<li type="circle">com.ibm.cuda.<a href="../../../com/ibm/cuda/CudaDevice.SharedMemConfig.html" title="enum in com.ibm.cuda"><span class="typeNameLink">CudaDevice.SharedMemConfig</span></a></li>
 <li type="circle">com.ibm.cuda.<a href="../../../com/ibm/cuda/CudaJitTarget.html" title="enum in com.ibm.cuda"><span class="typeNameLink">CudaJitTarget</span></a></li>
 <li type="circle">com.ibm.cuda.<a href="../../../com/ibm/cuda/CudaJitOptions.CacheMode.html" title="enum in com.ibm.cuda"><span class="typeNameLink">CudaJitOptions.CacheMode</span></a></li>
 <li type="circle">com.ibm.cuda.<a href="../../../com/ibm/cuda/CudaJitOptions.Fallback.html" title="enum in com.ibm.cuda"><span class="typeNameLink">CudaJitOptions.Fallback</span></a></li>
+<li type="circle">com.ibm.cuda.<a href="../../../com/ibm/cuda/CudaDevice.CacheConfig.html" title="enum in com.ibm.cuda"><span class="typeNameLink">CudaDevice.CacheConfig</span></a></li>
+<li type="circle">com.ibm.cuda.<a href="../../../com/ibm/cuda/CudaDevice.Limit.html" title="enum in com.ibm.cuda"><span class="typeNameLink">CudaDevice.Limit</span></a></li>
+<li type="circle">com.ibm.cuda.<a href="../../../com/ibm/cuda/CudaDevice.SharedMemConfig.html" title="enum in com.ibm.cuda"><span class="typeNameLink">CudaDevice.SharedMemConfig</span></a></li>
```

Comment:
All this reordering is disappointing: when I tried building the docs locally I got a order distinct from this and the previous content.